x-pack/plugins/enterprise_search/public/applications/workplace_search/routes.ts

```diff
@@ -114,3 +114,6 @@ export const getContentSourcePath = (
   sourceId: string,
   isOrganization: boolean
 ): string => generatePath(isOrganization ? ORG_PATH + path : path, { sourceId });
+export const getGroupPath = (groupId: string) => generatePath(GROUP_PATH, { groupId });
+export const getGroupSourcePrioritizationPath = (groupId: string) =>
+  `${GROUPS_PATH}/${groupId}/source-prioritization`;
```

Comment:
One more quick note (should've caught this earlier, sorry) - Kibana's convention for URLs is snake_casing, so we should change this to

```suggestion
  `${GROUPS_PATH}/${groupId}/source_prioritization`;
```

Same for all other instances of hyphenated URLs in this file - although you might want to do that as a separate PR